### PR TITLE
Create models for database and Weather station data upload

### LINKED ISSUE #26: Create Models for database
dependent on 

- [x] #22

### LINKED ISSUE #29: Fill weather station model with location data
- [x] Fill weather station model (weather stations with their location data from BC_Wildfire_Active_Weather_Stations.csv)

### PR DESCRIPTION
**Pull request by robertyac**
This PR completes 2 issues that I was working on. 

### Create models for database

I created the models based on the class diagram. I added and tested them out on the Django admin page and they seem to be working well. Test coverage Is over 80%

### Upload Weather station data

Here I wrote and tested a script that uploaded the csv data from the weather stations such as their longitude, latitude and elevation data. They all seem to be there and this script was tested with 100% coverage of this command.

## All tests located in the tests.py file 

Closes #26 
Closes #29

### CODE REVIEW

**Review by Davis245 — APPROVED** [review]
Looks good!

File: Site/bc_weather_station_dashboard/website/models.py [hunk 1]
```diff
@@ -57,4 +83,60 @@ class WeatherStation(models.Model):
...
+    ]
+    
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    theme = models.CharField(max_length=200, default='default')
+    layout = models.CharField(max_length=200, choices=LAYOUT_CHOICES, default='common')
+    
+    def __str__(self):
+        return self.user.username + "'s Dashboard Preferences"
+
+class Alert(models.Model):
```
Comment (cmcahoon01) [inline Site/bc_weather_station_dashboard/website/models.py:102]: May need to change depending on how we decide to implement alerts, for now, looks good

File: Site/bc_weather_station_dashboard/website/tests.py [hunk 2]
```diff
@@ -1,3 +1,235 @@
+# Create your tests here.
 from django.test import TestCase
```
Comment (cmcahoon01) [inline Site/bc_weather_station_dashboard/website/tests.py:2]: looks quite thorough 👌

**Review by cmcahoon01 — APPROVED** [review]
Well commented

**Comment by robertyac** [comment]
Yeah good point! I was looking at twilio for alerts (sms/email alerts) and
seems like a good way to go.

Even with the use of their sdk/api it may be a good idea to still store our
alerts in our own database but like you said we’ll figure it out once
we get there.

**Pull request merged**